src/views/check-your-appeal.njk

```diff
@@ -51,11 +51,11 @@
       <h2 class="govuk-heading-l">Reason for appeal</h2>
 
       {% set attachmentsList %}
```

Comment:
Check this?  Potentially looking at an array

Reply:
**attachmentsList** is used as variable for the supporting document where the html content created inside it is just an unordered list if `appealReasonDetails.attachments` exists otherwise a simple `None string.
Used as an alternative for macros to have a clean njk page